### PR TITLE
Provide two options to sanitize emojis (:simple and :generated regexes)

### PR DESCRIPTION
**Pull request by veelenga**
This PR addresses the performance issue discussed in https://github.com/veelenga/emoji.cr/pull/5

Now it is possible to sanitize emojis uses two options: `:simple` or `:generated` regex:

```crystal
Emoji.sanitize(str) # defaults to :simple
Emoji.sanitize(str, :simple)
Emoji.sanitize(str, :generated)
```

/cc @sija

### CODE REVIEW

File: src/emoji.cr [hunk 1]
```diff
@@ -15,7 +16,12 @@ module Emoji
     s
   end
 
-  def self.sanitize(s : String)
-    s.gsub(EMOJI_REGEX, "")
+  def self.sanitize(s : String, regex = :simple)
```
Comment (Sija) [inline src/emoji.cr]: Is it worth using enum here?

Comment (Sija) [reply src/emoji.cr]: Kind of off-topic but I'd replace `s` with `str` to be at least bit more descriptive ;)

Comment (veelenga) [reply src/emoji.cr]: I would prefer to keep it simple here. Regular symbol works just fine IMO

Comment (Sija) [reply src/emoji.cr]: With autocasting you still should be able to use them, having compile-time safety without manual checks :) but that's optional for me, LGTM in any case.

**Pull request merged**